internal/controlplane/handlers_githubwebhooks.go

```diff
@@ -681,7 +697,7 @@ func (s *Server) processRelevantRepositoryEvent(
 	if dbrepo.WebhookID.Valid {
 		// Check if the payload webhook ID matches the one we
```

Comment:
question: Is the check for `event.GetHookID() != 0` better suited to be in this `if` statement?

Reply:
You're right, I'll move it up.